pom.xml

```diff
@@ -622,6 +622,10 @@
               <!-- We also want to check the license in templates to generate files with the proper header -->
               <include>test/java/spoon/generating/clone/*</include>
             </includes>
+            <excludes>
+                <!--  excluding the code coming from Javaparser -->
+                <exclude>main/java/spoon/javadoc/**</exclude>                
```

Comment:
why the source in javadoc dir are excluded from checking of license header?

Reply:
ok, I see that license headers are different ... from javaparser

Reply:
Yes, we want to keep the original Javaparser double-license scheme.